views/apps/new.html

```diff
@@ -1,5 +1,52 @@
 {{define "body"}}
 
+{{if .User.HubUrl}}
+
+<p id="message" class="font-bold text-xl font-headline mb-4 dark:text-white text-center"></p>
+
+<script type="text/javascript">
+  if (!window.opener) {
+    // if there's no opener, redirect to hub url
+    document.getElementById("message").innerHTML = "Redirecting to Alby Hub..."
+    window.location.href = "{{.User.HubUrl}}/#/apps/new?{{.QueryString}}"
+  } else {
+    document.title = "Waiting for Alby Hub Connection...";
+    document.getElementById("message").innerHTML = `Please enable pop-ups and proceed in the new Alby Hub window.<br/>Keep this window open to receive updates.`
+
+    // open hub in another window
+    const hubWindow = window.open("{{.User.HubUrl}}/#/apps/new?{{.QueryString}}")
```

Comment:
can you check how we do this in bitcoin connect? - that we do it exactly the same. also with the same window size.

Reply:
We do it here: https://github.com/getAlby/bitcoin-connect/blob/master/src/components/connectors/bc-alby-nwc-connector.ts#L21

It comes from js-sdk: https://github.com/getAlby/js-sdk/blob/master/src/NWCClient.ts#L376-L387

So I even match the positions? Or shift some by x=10 and y=10?

Reply:
Done!